Aqovia.MultiplexingDistributedCache.sln

```diff
@@ -0,0 +1,36 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31903.59
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{6FAC0230-2D16-4669-974C-DB47ABC910C1}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MultiplexingDistributedCache", "src\MultiplexingDistributedCache\MultiplexingDistributedCache.csproj", "{18FB8F4C-E881-48F8-9E5C-267C2F2BB384}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{D73C110D-1798-45C8-B618-755986115812}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MultiplexingDistributedCache.Tests", "tests\MultiplexingDistributedCache.Tests\MultiplexingDistributedCache.Tests.csproj", "{B1CFAD9F-4F49-4958-BC79-110F5C042A51}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{18FB8F4C-E881-48F8-9E5C-267C2F2BB384}.Debug|Any CPU.ActiveCfg = Debug|Any CPU

```

Comment:
Does 'AnyCPU' only mean x86 and x64 or does it also cover ARM? Not a big deal if we can't solve it today but I'm keen that we ensure all our libraries are built for both ARM and intel architectures.

Reply:
I changed the target framework to .NET Standard 2.0, so more .NET runtimes support it.

.NET Standard is itself an API, not a runtime. It depends on the runtime to be able to run under ARM architecture, which most of the .NET runtime versions are able to. So yes, it can be used under ARM architecture. [Here](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0#select-net-standard-version) is the list of runtimes that support .Net Standard 2.0.